e2e/tests/targeting/predicated-variants/multiple-experiments.test.ts

```diff
@@ -0,0 +1,52 @@
+/// <reference path="../../../types.d.ts" />
+
+import { buildRequestHooks, getActiveKeys } from '../../../helpers';
+import { Page } from '../page.po';
+import hooks from './multiple-experiments.hooks';
+
+
+fixture `Predicated Variant Targeting > Multiple Experiments`
+    .page `http://localhost:9090/tests/targeting/index.html`
+    .requestHooks(
+    	buildRequestHooks(hooks)
+    );
+
+const page = new Page();
+
+test(`should handle multiple experiments`, async t => {
+	// Preconditions
+	await t
+		.expect(page.button.getStyleProperty('color'))
+		.eql('rgb(0, 0, 0)')
+		.expect(page.header.innerText)
+		.eql('Evolv')
+		.expect(page.button.getStyleProperty('background-color'))
+		.eql('rgb(239, 239, 239)')
+		.expect(page.button.innerText)
+		.eql('Click');
+
+	// Act
+	await t.eval(function() {
+		evolv.context.set('state', 'TX');
+		evolv.context.set('flavor', 'chocolate');
+	});
+
+	// Assert
+	const activeKeys = await getActiveKeys();
```

Comment:
As well as verifying the activeKeys can you verify the active variants in the context as well?